nomad/structs/structs.go

```diff
@@ -163,6 +163,9 @@ const (
 	DefaultNamespace            = "default"
 	DefaultNamespaceDescription = "Default shared namespace"
 
+	// DefaultNamespace is the default node pool.
+	DefaultNodePool = "default"
```

Comment:
I have this in `structs_node_pool.go` but with the name flipped 😅 
https://github.com/hashicorp/nomad/blob/45fcc59a82137bf69cb1f6087e5ce614c378cd34/nomad/structs/node_pool.go#L20-L23

Reply:
Hrm, we have it as `DefaultNodePool` in `api` in your [private branch](https://github.com/hashicorp/nomad-enterprise/blob/f-node-pools-nodes-api/api/node_pools.go#L8) which is why I did that here. It also matches `DefaultNamespace`. We should probably be consistent between `api` and `nomad/structs` just so we don't frustrate ourselves. Which would you prefer?

Reply:
Eh, because there's also a `NodePoolAll` it makes for nicer autocomplete to have it be `NodePoolDefault`. Will fix.

Reply:
Ah sorry, I didn't notice the inconsistency. Yeah, I think `NodePoolDefault` and `NodePoolAll` work better. I reconciled their use in https://github.com/hashicorp/nomad/pull/17388.